fitnessllm_dataplatform/batch_handler.py

```
@@ -17,23 +22,67 @@
 
+        if data_source == FitnessLLMDataSource.STRAVA.value:
+            strava_refresh_token = (
+                user_ref.collection("stream")
+                .document(data_source.lower())
+                .get()
+                .to_dict()
+                .get("refreshToken")
+            )
+            assert strava_refresh_token
+            strava_refresh_oauth_token(
+                db=self.db, uid=user_id, refresh_token=strava_refresh_token
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)